frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -106,6 +107,13 @@ export const StrategyItemContainer: FC<IStrategyItemContainerProps> = ({
     const Icon = getFeatureStrategyIcon(strategy.name);
     const { uiConfig } = useUiConfig();
 
+    // @ts-expect-error IFeatureStrategy doesn't have links
```

Comment:
I would solve it without ignoring typechecker. When I checked out your branch none of the union elements had a links field

Reply:
AFter we get the links on one of the union element we can play around with type aliases and ternany operators so that we don't have to opt out of type safety

Reply:
Fair! I'll rebase off main now that the Orval updates have been merged and see if that fixes it.